.goreleaser.yaml

```diff
@@ -0,0 +1,248 @@
+
+changelog:
+  disable: >-
+    {{ if eq (index .Env "SKIP_SCM_RELEASE") "true" }}{{true}}{{ else }}{{false}}{{ end }}
+  sort: asc
+  use: github
+  format: "{{ .SHA }}: {{ .Message }}{{ with .AuthorUsername }} by @{{ . }}{{ end }}"
+  filters:
+    exclude:
+      - "^test:"
+      - "^test\\("
+      - "^(build|ci): "
+      - "^ci\\("
+      - "^chore:"
+      - "^chore\\("
+      - "merge conflict"
+      - "merge conflict"
+      - Merge pull request
+      - Merge remote-tracking branch
+      - Merge branch
+  groups:
+    - title: "🚀 New Features 🚀"
+      regexp: '^.*?(feat|enh|enhancement)(\([[:word:]]+\))??!?:.+$'
+      order: 100
+    - title: "🔐 Security updates 🔐"
+      regexp: '^.*?sec(\([[:word:]]+\))??!?:.+$'
+      order: 150
+    - title: "🐛 Notable Fixes 🐛"
+      regexp: '^.*?fix(\([[:word:]]+\))??!?:.+$'
+      order: 200
+    - title: "📄 Documentation updates 📄"
+      regexp: ^.*?docs?(\([[:word:]]+\))??!?:.+$
+      order: 300
+    - title: "✨ Notable Changes ✨"
+      order: 9999
+
+release:
+  github:
+    owner: k0rdent
+    name: kcm
+  draft: true
+  replace_existing_draft: true
+  replace_existing_artifacts: true
+  prerelease: auto
+  make_latest: true
+  mode: replace
+
+  disable: >-
+    {{ if eq (index .Env "SKIP_SCM_RELEASE") "true" }}{{true}}{{ else }}{{false}}{{ end }}
+
+  header: |
+    ## 📋 Components Versions 📋
+
+    | Provider Name | Version |
+    | --- | --- |
+    | Cluster API | {{ .Env.CHART_CLUSTER_API_VERSION }} |
+    | Cluster API Provider AWS | {{ .Env.CHART_CLUSTER_API_PROVIDER_AWS_VERSION }} |
+    | Cluster API Provider Azure | {{ .Env.CHART_CLUSTER_API_PROVIDER_AZURE_VERSION }} |
+    | Cluster API Provider Docker | {{ .Env.CHART_CLUSTER_API_PROVIDER_DOCKER_VERSION }} |
+    | Cluster API Provider GCP | {{ .Env.CHART_CLUSTER_API_PROVIDER_GCP_VERSION }} |
+    | Cluster API Provider Infoblox | {{ .Env.CHART_CLUSTER_API_PROVIDER_INFOBLOX_VERSION }} |
+    | Cluster API Provider IPAM | {{ .Env.CHART_CLUSTER_API_PROVIDER_IPAM_VERSION }} |
+    | Cluster API Provider k0smotron | {{ .Env.CHART_CLUSTER_API_PROVIDER_K0SPROJECT_K0SMOTRON_VERSION }} |
+    | Cluster API Provider OpenStack (ORC) | {{ .Env.CHART_CLUSTER_API_PROVIDER_OPENSTACK_VERSION }} ({{ .Env.CHART_ORC_VERSION }}) |
+    | Cluster API Provider vSphere | {{ .Env.CHART_CLUSTER_API_PROVIDER_VSPHERE_VERSION }} |
+    | Projectsveltos | {{ .Env.CHART_PROJECTSVELTOS_VERSION }} |
+
+    ---
+
+  footer: |
+    ---
+    **Full Changelog**: https://github.com/k0rdent/kcm/compare/{{ .PreviousTag }}...{{ .Tag }}
+
+  extra_files:
+    - glob: ./install.yaml.out
+      name_template: install.yaml
+    - glob: ./templates/provider/kcm-templates/files/release.yaml
+
+  # Upload metadata.json and artifacts.json to the release as well.
+  include_meta: true
+
+dockers:
+  - id: linux-amd64
+    goos: linux
+    goarch: amd64
+    image_templates:
+      - "{{ .Env.REGISTRY }}/{{ .Env.IMAGE_NAME }}:{{ .Env.VERSION }}-amd64"
```

Comment:
With this change, we build the controller image with an arch suffix (`-amd64`, `-arm64v8` in the tag, right? But in the release job, we still use 
```
run: VERSION="${{ steps.version.outputs.version }}" make kcm-chart-release
```
so, in the kcm helm chart, we'll have the default controller image defined without any arch in tag `ghcr.io/k0rdent/kcm/controller:${{ VERSION }}`. Will this image be present in the artifactory?
Or am I misunderstanding something?

Reply:
It will, thanks to docker manifests. The umbrella image (the one without architecture postfix) will have 3 images in it: amd64v1, arm64v8, and armv7

Effectively, it is all done within the `docker_manifests` section (there is a bug, though, which I've already fixed in the upstream; however, the new version is not yet published, hence it is now not possible to build images for only one arch)